bin/prover-client/Cargo.toml

```diff
@@ -41,6 +41,7 @@ rayon = "1.8.0"
 reth-rpc-types = { workspace = true }
 rockbound = { workspace = true }
 serde = { workspace = true }
+serde_json = { workspace = true }
```

Comment:
Can we change all this `foo = { workspace = true }` to `foo.workspace = true` in this file to match our root and other `Cargo.toml`s in the repo?

Reply:
What's the preferred convention for enabling workspace dependencies: using the form `foo = { workspace = true }` versus `foo.workspace = true`? Are there any specific guidelines or best practices to follow?

I remember @delbonis asking to use `foo = { workspace = true }`.

Reply:
We changed this when we renamed to `strata`. See the root-level `Cargo.toml` and a bunch of others `Cargo.toml`s on `main`.
Not putting words into Trey's mouth but I think he said that we should do whatever we want but be consistent.

Since the `foo.workspace = true` is more succinct, we are using that one. Now the only `Cargo.toml` that I could find that is inconsistent is this one.